ui/public/mcp.js

```diff
@@ -4,8 +4,8 @@ console.log('Mobile Control Panel Extension Loaded');
 window.OPENSHIFT_CONSTANTS.PROJECT_NAVIGATION.splice(1, 0, {
```

Comment:
Yeah, good point.
I've been holding out on giving it too much attention until it can be split up into individual files i.e. https://issues.jboss.org/browse/FH-3826